### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dmwinslow**
Update with work-in-progress disclaimer.

### CODE REVIEW

**Comment by dmwinslow** [comment]
@hydrocrust feel free to review the specific text in "files changed" and either approve or comment - could give you a feel for how reviews work.

**Comment by hydrocrust** [comment]
I am not certain how to "review." I opened README.md and made some edits. I changed a single sentence describing FEHM/Lagrit and the purpose of the Toolkit. Maybe Dustin can walk me through this process so that I'll better understand how to edit or request changes to text. Not sure what is best way to do this and retain tracking of file contents, etc.

**Review by hydrocrust — COMMENTED** [review]
Aha, I found where to add a comment to the Files changed tab. 
Looks like I could now choose Approve and it would "approve merging these changes." Not sure if this would include the changes I just made, or the changes made by Dustin before I got here. Will ask tomorrow when we meet. :)

**Comment by hydrocrust** [comment]
Hmm, I see a link above for Github desktop. Should I (we) be using that?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # FEHM Toolkit
-This toolkit consists of utilities and convenience functions to enable fast and efficient use of [FEHM](https://github.com/lanl/FEHM) and [LaGriT](https://github.com/lanl/LaGriT), which are software packages created by Los Alamos National Labs for hydrothermal simulations.
+This toolkit consists of utilities and convenience functions to enable fast and efficient use of [FEHM](https://github.com/lanl/FEHM) and [LaGriT](https://github.com/lanl/LaGriT), which are software packages created by Los Alamos National Labs for simulations of coupled fluid-heat-solute transport and related processes. The specific focus of the FEHM Toolkit is on simulations of seafloor hydrothermal circulation.   
+
+## Work in progress
```
Comment (dmwinslow) [inline README.md:4]: Let's call this "under construction" instead.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+This toolkit is in the early stages of development and is not yet in working order. This page will be updated when an initial release is ready. Work to date is focused on collating and testing previous Matlab versions of the code.
```
Comment (dmwinslow) [inline README.md:5]: Let's change the wording.

**Review by dmwinslow — COMMENTED** [review]
Looks good, but see inline.

**Pull request merged**